pkg/controller/installation/core_controller.go

```diff
@@ -1404,6 +1419,41 @@ func (r *ReconcileInstallation) setDefaultsOnFelixConfiguration(ctx context.Cont
 	return nil
 }
 
+var osExitOverride = os.Exit
+
+// checkActive verifies the operator that calls this function is designated as the active operator.
+// If this operator is not designated as active then this function does an os.Exit(0) so the operator
+// gets restarted.
+// If this operator is the designated operator (or assumed because there is no designation) then
+// this function returns with no error.
+// If the active operator designation needs to be set then the first return field is a ConfigMap that
+// should be created to set the designation, other wise the field is nil.
+// The second returned field reports if there was an error when trying to determine active operator.
+func (r *ReconcileInstallation) checkActive(log logr.Logger) (*corev1.ConfigMap, error) {
+	cm, err := active.GetActiveConfigMap(r.client)
+	if err != nil {
+		r.SetDegraded(
```

Comment:
If multiple operators hit this, I'm guessing they will fight with each other to maintain this active operator status?

Reply:
It would only be an initial race. Both (or multiple) if running at the same time one could create the map and then a second could overwrite the map. But once either re-reads the config they would not touch it again since they only create the configMap if it doesn't already exist.
So I wouldn't say they fight because an operator will only attempt to create this ConfigMap once.

Reply:
I meant if two or more operators fail to get the active configmap, they will both try to set degraded on the same tigera status resource.

Probably not an issue? Will they try to do anything differently from each other?

Reply:
Oh I see. Sorry about misunderstanding that.
Yeah I see what you mean. Maybe we should not SetDegraded in this specific case and just log it? Or because of the possible fight should we include the namespace in the message, that way it would be an indicator if it was coming from an unexpected operator?